receiver/simpleprometheusreceiver/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simpleprometheusreceiver
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config defines configuration for simple prometheus receiver.
+type Config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"`
+	// CollectionInterval is the interval at which metrics should be collected
+	CollectionInterval time.Duration `mapstructure:"collection_interval"`
+	// MetricsPath the path to the metrics endpoint.
+	MetricsPath string `mapstructure:"metrics_path"`
+	// Whether or not to use pod service account to authenticate.
+	UseServiceAccount bool `mapstructure:"use_service_account"`
+	// TLS configs in case connection is over HTTPS
+	TLSCredentials *tlsCredentials `mapstructure:"tls_credentials"`
+}
+
+// TODO: Move to a common package for use by other receivers and also pull
+// in other utilities from
+// https://github.com/signalfx/signalfx-agent/blob/master/pkg/core/common/httpclient/http.go.
+type tlsCredentials struct {
+	// Path to the CA cert that has signed the TLS cert.
+	CAFilePath string `mapstructure:"ca_file"`
+	// Path to the client TLS cert to use for TLS required connections.
+	CertFilePath string `mapstructure:"cert_file"`
+	// Path to the client TLS key to use for TLS required connections.
+	KeyFilePath string `mapstructure:"key_file"`
```

Comment:
Maybe remove Path from the variable names since they got removed from the config names as well?